pkg/generator/templates/cloud-init-ubuntu.template

```diff
@@ -1,60 +1,65 @@
-#cloud-config
-apt_update: false
-manage_etc_hosts: true
-write_files:
-{{ if .Bootstrap -}}
-- path: '/etc/cloud/cloud.cfg.d/custom-networking.cfg'
-  permissions: '0644'
-  encoding: b64
-  content: |
-    bmV0d29yazoKICBjb25maWc6IGRpc2FibGVkCg==
-{{ end -}}
-{{ if and (isContainerDEnabled .CRI)  .Bootstrap -}}
-- path: '/etc/systemd/system/containerd.service.d/11-exec_config.conf'
-  permissions: '0644'
-  encoding: b64
-  content: |
-    W1NlcnZpY2VdCkV4ZWNTdGFydD0KRXhlY1N0YXJ0PS91c3IvYmluL2NvbnRhaW5lcmQgLS1jb25maWc9L2V0Yy9jb250YWluZXJkL2NvbmZpZy50b21s
-{{ end -}}
+#!/bin/bash
```

Comment:
I compared this file with the corresponding file in `gardener-extension-os-ubuntu-alicloud` and at first glance there seems to be quite a number of differences. It seems it's not immediately possible to archive the `ubuntu-alicloud` extension, at least not without further rework and testing. Are you aware of any major issues?

Reply:
I think the major difference is the `PROVIDER_ID` env variable that is already [injected](https://github.com/gardener/gardener-extension-provider-alicloud/blob/a8ccd95d46bf9d51ce331ef0b60fa97f19cb3496/pkg/webhook/controlplane/ensurer.go#L101-L127) by the provider-alicloud.